application/values.yaml

```diff
@@ -672,7 +672,7 @@ externalSecret:
 pdb:
   enabled: false
   minAvailable: 1
-  maxUnavailable: 2
+  maxUnavailable:
```

Comment:
@fahad-rauf why this change?

Reply:
to show that one of them can be null. What do you think should be default because if both are available min will be used anyways.

Reply:
yeah we should set to some value; maybe we can set to same value? i don't know what is the best practice; i would google and learn

Reply:
I could not find any best practice in helm regarding how values file should be when a field is nullable. I think its a choice of what feels best. I think i will add some int value so people know what it could be.

Reply:
@fahad-rauf  you can comment both fields i.e max and min. people can uncomment and use whatever they want

Reply:
we need one field to make this work so i will comment out maxUnavailable